riemann_client/client.py

```diff
@@ -152,6 +153,9 @@ def create_dict(event):
                 value = list(value)
             elif descriptor.name == 'attributes':
                 value = dict(((a.key, a.value) for a in value))
+            elif descriptor.type == \
+                    google.protobuf.descriptor.FieldDescriptor.TYPE_FLOAT:
+                value = round(value, 1)
```

Comment:
Might be better to remove the test this breaks - I'm a little worried that this could break things in suprising ways for other users.